redhat-operators/tektoncd-pipeline-operator/tektoncd-pipeline-operator.package.yaml

```diff
@@ -0,0 +1,5 @@
+packageName: tetektoncd-pipeline-operatorktoncd
+kind: PackageManifest
```

Comment:
nit: `package.yaml` file does not use `kind` field, please remove it.

Reply:
done